Trappist/src/Promact.Trappist.Test/Category/CategoryRepositoryTest.cs

```diff
@@ -87,14 +100,18 @@ public CategoryRepositoryTest(Bootstrap bootstrap) : base(bootstrap)
         [Fact]
         public async Task DeleteCategory()
         {
-            var category = CreateCategory();
-            await _categoryRepository.AddCategoryAsync(category);
-            var categoryToDelete = await _categoryRepository.GetCategoryByIdAsync(category.Id);
+            var applicationUser = await CreateUserAsync("karan.mehta@promactinfo.com");
+            var firstCategory = await CreateCategoryAsync("History");
+            var SecondCategory = await CreateCategoryAsync("General knowledge");
+            var singleAnswerQuestion = CreateSingleAnswerQuestionAsync(SecondCategory.Id, applicationUser.Id);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Remove this unused 'singleAnswerQuestion' local variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://server12:9005/coding_rules#rule_key=csharpsquid%3AS1481)